### PR TITLE
Stage submitter realification

### PR DESCRIPTION
**Pull request by jdotpz**

### CODE REVIEW

File: puppet/modules/socorro/manifests/role/stagesubmitter.pp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      ];
+
+    'socorro-stagesubmitter':
+      ensure  => running,
+      enable  => true,
+      require => Exec['join_consul_cluster'];
+  }
+
+    '/etc/nginx/conf.d/socorro-stagesubmitter.conf':
+      source  => 'puppet:///modules/socorro/etc_nginx/conf_d/socorro-stagesubmitter.conf',
```
Comment (jdotpz) [inline puppet/modules/socorro/manifests/role/stagesubmitter.pp]: Even though we don't use nginx to access anything regarding this part of the app, our standard is service files in this dir.

Comment (jdotpz) [inline puppet/modules/socorro/manifests/role/stagesubmitter.pp]: Squashed the commits.

File: puppet/modules/socorro/manifests/role/stagesubmitter.pp [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  }
+
+  file {
+    '/etc/nginx/conf.d/socorro-stagesubmitter.conf':
+      source  => 'puppet:///modules/socorro/etc_nginx/conf_d/socorro-stagesubmitter.conf',
+      owner   => 'root',
+      group   => 'nginx',
+      mode    => '0664',
+      require => File['/etc/nginx/nginx.conf']
+      }
```
Comment (phrawzty) [inline puppet/modules/socorro/manifests/role/stagesubmitter.pp]: `}` is improperly indented. 🐐

File: puppet/modules/socorro/manifests/role/stagesubmitter.pp [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+class socorro::role::stagesubmitter {
+
+include socorro::role::common
+
+  service {
+    'nginx':
+      ensure    => running,
+      enable    => true,
+      subscribe => File[
+        '/etc/nginx/conf.d/socorro-stagesubmitter.conf'
```
Comment (phrawzty) [inline puppet/modules/socorro/manifests/role/stagesubmitter.pp]: IIRC, the style that is used in this repo prefers putting single-item arrays on one line.

In other words, compress lines 11 through 12. 🐄

File: terraform/stagesubmitter/main.tf [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        "${var.region}c"
+    ]
+    depends_on = [
+        "aws_launch_configuration.lc-stagesubmitter"
+    ]
+    launch_configuration = "${aws_launch_configuration.lc-stagesubmitter.id}"
+    max_size = 1
+    min_size = "${lookup(var.stagesubmitter_num, var.environment)}"
+    desired_capacity = "${lookup(var.stagesubmitter_num, var.environment)}"
+    tag {
```
Comment (phrawzty) [inline terraform/stagesubmitter/main.tf]: All of the `tag` blocks appear to use two-space indentation, when the internal style in this repo appears to prefer four-space indents.

That said, I think this "mistake" is replicated across *all* of the `main.tf`s, so it probably makes sense to replicate it here for the sake of consistency. 🚯

**Comment by phrawzty** [comment]
As long as there aren't any collisions, it might make sense to merge `master` in (or at least a50ce79) so as to let Travis have a proper crack at it.

**Comment by jdotpz** [comment]
I've merged in your changes from master and travis-ci is now happy.
Thanks for the eyes, I'll get those styling changes done!

**Comment by phrawzty** [comment]
`r+` with the following conditions:

1. Those two-space tag sections 😿 
1. I am not qualified to review `socorro-config/stagesubmitter.conf` and therefore absolve myself of all responsibility therein.

**Comment by jdotpz** [comment]
I am good with those conditions.
Sometimes, when a bug gets to prod, it becomes a feature! :) (referring to the two-space tag sections)

**Pull request merged**